documentation/src/main/asciidoc/user_guide/hibernate-ogm.adoc

```diff
@@ -0,0 +1,47 @@
+==  Hibernate OGM
+Hibernate can perform CRUD operations directly on an Infinispan cluster.
+Hibernate OGM is an extension of the popular Hibernate ORM project which makes
+the Hibernate API suited to interact with NoSQL databases such as Infinispan.
+
+From the perspective of the Hibernate user, it might be interesting to replace
```

Comment:
This sentence sounds uncertain, maybe you can change `it might be interesting to replace the RDBMS` with something more similar to `Infinispan offers better scalability and higher performance on CRUD ... compared to an RDBMS `